### PR TITLE
Update Dependencies, notably Alexandria and ESLint

### PR DESCRIPTION
**Pull request by camjc**
- add webpack-sources as dev dep until webpack/extract-text-webpack-plugin#142
- Clean up scss and examples for Button.
- Remove :active outlines created by bootstrap.
- Run JSCS in `posttest`

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -31,37 +32,41 @@ class AppComponent extends React.Component {
       <div className="index">
         <h1>Buttons</h1>
         <div className="btn-panel">
-          <Button className="btn-inverse">
-            Default
+          <Button>
```
Comment (etaoins) [inline src/components/Main.js]: Should this be `<Button className="btn-inverse">`?

**Comment by etaoins** [comment]
LGTM :green_apple:

File: src/components/Main.js (hunk 1 above)
Comment (camjc) [inline src/components/Main.js]: Yeah probably. Do you think its wise to enforce no default button by making default button look wrong?

**Comment by robmunro** [comment]
Merging

**Pull request merged**